SimplioOfficial/simplio-app
# Feature/initial infrastructure

## Overview

Setup development infrastructure, continuous integration and deployment. Build an initial flutter scaffold project with selected tech stack.

### Test cases:

1. Open wallet and control if there are no wallets created. There should be a text `You have no wallets` in the center. 
2. Click on a plus symbol in top right corner. A page with named wallets and toggle switches should appear.
3. Create a wallet by switching on a wallet.
4. Go back and check if created wallet is listed.
5. Click the plus symbol in top right corner again and switch off the created wallet.
6. Go back and check if there is no wallet and text in the center `You have no wallets`.
